### DISCUSSION TITLE
23.10.30 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#47] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 소셜 로그인 테스트 코드 작성 (Composition over Inheritance)
- 내일은 어떤 일을 하나요?
    - 테스트 코드 에러 핸들링..😵

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 코드 리뷰 반영
- 내일은 어떤 일을 하나요?
    - 노래 등록
    - 플레이리스트에 노래 추가

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 비동기 공부
- 내일은 어떤 일을 하나요?
  - 비동기 수정 작업 Issue, PR 작성
